PageObjects/OrderPage.cs

```diff
@@ -47,16 +48,20 @@ public void ChangeQty(string name, int qty)
             {
                 Driver.GetElement(ItemAddQty(name)).Click();
             }
-            Driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(5);
+            Thread.Sleep(TimeSpan.FromSeconds(1));          
         }
 
         public void CheckTotalPrice(string expectedTotalPrice)
         {
-            var actualTotalPrice = Driver.GetElement(CurrentTotalPrice).Text.Trim('$');
+            var actualPrice = new ElementLocator(Locator.Id, "total_price");
+            var actualTotalPrice = Driver.GetElement(actualPrice).Text.Trim('$');
             Assert.That(actualTotalPrice, Is.EqualTo(expectedTotalPrice));
         }
 
-        public ElementLocator CurrentTotalPrice => new ElementLocator(Locator.Id, "total_price");
+        public ElementLocator GetCurrentTotalPrice()
+        {
+            return new ElementLocator(Locator.Id, "total_price");
```

Comment:
lokatory trzymamy na górze

Reply:
to metodę przenieść na samą górę? czy jak? w momencie wywołania metody chcę pobrać wartość - nie w momencie gdy następuje inicjalizacja obiektu

Reply:
fixed